src/lib/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, ComponentProps, PropsWithChildren } from 'react';
+import { HiOutlineChevronRight } from 'react-icons/hi';
+
+export interface BreadcrumbItemProps extends ComponentProps<'li'> {
+  href?: string;
+  icon?: FC<ComponentProps<'svg'>>;
+  isFirst?: boolean;
+  isLast?: boolean;
+}
+
+const BreadcrumbItem: FC<PropsWithChildren<BreadcrumbItemProps>> = ({
+  children,
+  href,
+  icon: Icon,
+  isFirst,
+  isLast,
+}): JSX.Element => {
+  return (
+    <li className="flex items-center">
+      {!isFirst && (
+        <HiOutlineChevronRight
+          aria-hidden="true"
+          className="mx-1 h-6 w-6 text-gray-400 md:mx-2"
+          data-testid="breadcrumb-separator"
+        />
+      )}
+      {!isLast ? (
```

Comment:
I think even the last one can be a link, so this should be tested against if the `href` prop is defined or not.

Reply:
In that case, I think this can be simplified by removing the reference to `isLast` and simply checking for an `href`.